portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,67 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+//Training of fecth()
+async function addGreetingElmer(){
+
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.text();
+
+    const dateContainer = document.getElementById('greeting-container');
+    dateContainer.innerText = textFromResponse;
+    
+}
+/** Fetches stats from the server and adds them to the page. */
+async function getRandomFactAboutMeEN(){
+    const responseFromServer = await fetch('/randomFactsAboutMe');
+    const listFacts = await responseFromServer.json();
+    const factContainer = document.getElementById('fact-container');
+    console.log(listFacts.length);
+    
+    const fact = listFacts.factsInEN[Math.floor(Math.random() * (listFacts.length-1))];
+    
+    factContainer.innerHTML='';
+    factContainer.appendChild(
+        createImagesElement(fact.pathImage, fact.altAttribute)
+    );
+
+    factContainer.appendChild(
+        createParagraphElement(fact.factText)
+    );
+    
```

Comment:
nit: I recommend using the cloud shell formatter to format your files so they are more readable :)

In cloud shell you can right click and select format this document:
![image](https://user-images.githubusercontent.com/34181040/122113223-a6cbc680-cdd6-11eb-9f0f-0e065489d348.png)